src/eigenlayer/ObolEigenLayerPodController.sol

```diff
@@ -0,0 +1,176 @@
+    address public split;
+
+    /// @notice address of owner
+    address public owner;
+
+
+    modifier onlyOwner() {
+        if (msg.sender != owner) {
+            revert Unauthorized();
+        }
+        _;
+    }
+
+    constructor(
+        address recipient,
+        uint256 share,
+        address delegationManager,
+        address eigenPodManager,
+        address withdrawalRouter
+    ) {
+        if (recipient != address(0) && share == 0) revert Invalid_FeeSetup();
+
+        feeRecipient = recipient;
+        feeShare = share;
+        eigenLayerDelegationManager = delegationManager;
+        eigenLayerPodManager = IEigenPodManager(eigenPodManager);
+        delayedWithdrawalRouter = IDelayedWithdrawalRouter(withdrawalRouter);
+    }
+
+    /// @dev Enables contract to receive ETH
+    // defined on the clone implementation
+    // receive() external payable {}
+
+    /// @notice initializes the controller
+    /// @param _owner address of the controller owner
+    /// @param splitter address of splitter
+    function initialize(address _owner, address splitter) external {
+        if (owner != address(0)) revert AlreadyInitialized();
+
+        eigenPod = eigenLayerPodManager.createPod();
+        owner = _owner;
+        split = splitter;
+
+        emit Initialized(eigenPod, _owner);
+    }
+
+    /// @notice Call the eigenPod contract
+    /// @param data to call eigenPod contract
+    function callEigenPod( 
+        bytes calldata data
+    ) external payable onlyOwner {
+        _executeCall(address(eigenPod), msg.value, data);
+    }
+
+    /// @notice Call the Eigenlayer delegation Manager contract
+    /// @param data to call eigenPod contract
+    function callDelegationManager(
+        bytes calldata data
+    ) external payable onlyOwner {
+        _executeCall(address(eigenLayerDelegationManager), msg.value, data);
+    }
+
+    /// @notice Call the Eigenlayer Manager contract
+    /// @param data to call contract
+    function callEigenPodManager(
+        bytes calldata data
+    ) external payable onlyOwner {
+        _executeCall(address(eigenLayerPodManager), msg.value, data);
+    }
+
+    /// @notice Withdraw funds from the delayed withdrawal router
+    /// @param numberOfDelayedWithdrawalsToClaim number of claims
+    function claimDelayedWithdrawals(uint256 numberOfDelayedWithdrawalsToClaim)
+        external
+    {
+        delayedWithdrawalRouter.claimDelayedWithdrawals(
+            address(this),
+            numberOfDelayedWithdrawalsToClaim
+        );
+
+        // transfer eth to split
+        uint256 balance = address(this).balance;
+        if (feeShare > 0) {
+            uint256 fee =  (balance * feeShare) / PERCENTAGE_SCALE;
+            feeRecipient.safeTransferETH(fee);
+            split.safeTransferETH(balance -= fee);
+        } else {
+            split.safeTransferETH(address(this).balance);
+        }
+    }
+
+    /// @notice Rescue stuck tokens
```

Comment:
```suggestion
    /// @notice Rescue stuck tokens by sending them to the split contract.
```

Should we consider something `onlyOwner` that instead can go to an arbitrary address? Maybe wondering if it matters (e.g. a rebase token? is that the only danger?). This contract isn't directly the withdrawal address for a validator right? the odds it gets airdropped tokens should be relatively low I guess? Probably don't have to worry.